hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CSQueue.java

```diff
@@ -90,7 +90,7 @@
   public String getQueuePath();
 
   /**
-   * Get the object of the queue.
+   * Gets the object of the queue.
```

Comment:
Nit: Gets the queue path object. 

The object of the queue suggests a CSQueue object.

Reply:
Will fix the nit upon commit if I'm fine with the whole patch. Thanks for noticing.